README.md

```diff
@@ -107,60 +107,71 @@ CLI Options:
   -v, --version    Show the version
 
 Beautifier Options:
-  -s, --indent-size                 Indentation size [4]
-  -c, --indent-char                 Indentation character [" "]
-  -e, --eol                         character(s) to use as line terminators. (default newline - "\\n")');
+  -s, --indent-size             Indentation size [4]
```

Comment:
Because I didn't think it was broken haha.  My understanding of the pattern was a loosely tiered "match your neighbor's indentation".  I take it you don't want the options under "CLI Options" to be pushed over, just those under "Beautifier Options"?